### PR TITLE
Implemented missing cat endpoints

### PR DESCRIPTION
**Pull request by Mpdreamz**
* cat snapshots
* cat nodeattrs
* cat help
* cat repositories

Should backport this to 2.x

Also fixes some code generation path issues now the projects live under the same folder and the output for the classic projects are under `bin\Net45\` instead of `bin`

### CODE REVIEW

File: src/CodeGeneration/CodeGeneration.LowLevelClient/ApiEndpoints/cat.snapshots.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
       "parts": {
         "repository": {
           "type" : "list",
+          "required" : true,
```
Comment (Mpdreamz) [inline src/CodeGeneration/CodeGeneration.LowLevelClient/ApiEndpoints/cat.snapshots.json]: have to send a PR in the main repos for this: https://github.com/elastic/elasticsearch/issues/17216

File: src/Nest/Cat/CatNodeAttributes/CatNodeAttributesRequest.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+#pragma warning disable 612
+
+namespace Nest
+{
+	[Obsolete()]
```
Comment (gmarz) [inline src/Nest/Cat/CatNodeAttributes/CatNodeAttributesRequest.cs]: :+1:

**Comment by gmarz** [comment]
LGTM

File: src/Nest/Cat/CatNodeAttributes/CatNodeAttributesRequest.cs (hunk 2 above)
Comment (Mpdreamz) [inline src/Nest/Cat/CatNodeAttributes/CatNodeAttributesRequest.cs]: ohh I should probably add a better message there ?

**Comment by gmarz** [comment]
Backported to 2.x via e2c990f382ef7fc3a6e7af323b019c2afa3a708b

**Pull request merged**